python/pyspark/ml/regression.py

```diff
@@ -743,6 +743,18 @@ def treeWeights(self):
         """Return the weights for each tree"""
         return list(self._call_java("javaTreeWeights"))
 
+    @property
```

Comment:
This question applies to Scala API as well. Partly, I think, because in the Scala API things were done incrementally. But also because there may be some hope in the future that more predictors provide feature importances, and then it could be an abstract method in the `Predictor` class instead. I don't know that there's a better reason.